src/js/migration.js

```diff
@@ -61,9 +61,6 @@ document.addEventListener("DOMContentLoaded", async function () {
 
   // Update contextual menu.
   chrome.runtime.sendMessage({ action: "updateContextualMenu" });
-
-  // Redirect to welcome URL.
-  window.open(MiscURLs.INSTALLED_URL, "_self");
```

Comment:
Beside don't opening the welcome screen you should close the current tab (otherwise the tab will show the "migrating" message indefinitely).